### PR TITLE
feat: create verified reviews loader and fix types

### PR DESCRIPTION
**Pull request by pedrobernardina**
## What is this contribution about?

Although the platform offers an extension to the PDP Loader to include Verified Reviews, it offers no way of sorting or fetching new reviews after the initial render. This PR creates a loader to be invoked on client side to fetch reviews on demand.

It also fixes some types.

A local version of both #634  and this PR can be seen in the preview bellow. 

https://deco-sites-simples-f7ec095wncm9.deno.dev/camiseta-manga-curta-algodao-peruano-branca-0077065014/p

### CODE REVIEW

**Review by gabrielMatosBoubee — CHANGES_REQUESTED** [review]

File: verified-reviews/utils/client.ts [hunk 1]
```diff
@@ -11,10 +15,10 @@ export interface PaginationOptions {
...
   order?:
     | "date_desc"
-    | "date_ASC"
-    | "rate_DESC"
-    | "rate_ASC"
-    | "helpfulrating_DESC";
+    | "date_asc"
+    | "rate_desc"
+    | "rate_asc"
+    | "most_helpful";
```
[the remark this answers is not in the document]
Comment (guitavano) [reply verified-reviews/utils/client.ts:21]: Yes, @pedrobernardina can you keep older version and convert inside the function? We may have some stores with "date_ASC" saved on our panel, for example.

**Comment by matheusgr** [comment]
@pedrobernardina can you reply the comments from @gabrielMatosBoubee ?

**Comment by guitavano** [comment]
UP @pedrobernardina

**Comment by pedrobernardina** [comment]
Hi guys, sorry for the (really) late reply. I'll check these comments and resume the work on this PR soon.

**Comment by pedrobernardina** [comment]
@guitavano @gabrielMatosBoubee 

I'm closing this one because it's already included in #634 
I'll link this PR there to keep track of review comments.

**Pull request closed without merging**